### PR TITLE
Adds projext sevenzip (lzma SDK)

### PR DESCRIPTION
**Pull request by catenacyber**
For the moment, I use my own repo with some modifications to 7zip lzma SDK source code.
What do you think of this ?

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
Why not call this "7zip" as the project seems to be named instead of "sevenzip"

**Comment by jonathanmetzman** [comment]
Can you see why bad build check fails: https://api.travis-ci.org/v3/job/534870426/log.txt

File: projects/sevenzip/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 Google Inc.
```
Comment (jonathanmetzman) [inline projects/sevenzip/Dockerfile]: nit: 2019

File: projects/sevenzip/build.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -eu
+# Copyright 2018 Google Inc.
```
Comment (jonathanmetzman) [inline projects/sevenzip/build.sh]: nit: 2019

**Comment by catenacyber** [comment]
>Why not call this "7zip" as the project seems to be named instead of "sevenzip"

I had a previous naming problem with another project :
https://github.com/google/oss-fuzz/pull/1490#issuecomment-395563438
Is it fixed ?

> Can you see why bad build check fails: https://api.travis-ci.org/v3/job/534870426/log.txt

Sorry. I fixed it from project (cmake debug left over)

> How is this project different from this one: https://github.com/google/oss-fuzz/tree/master/projects/lzma

cc @fancycode
It is the same :-/
It was not here when I started
So I guess that we can close this...

**Pull request closed without merging**